src/tests/test_magic_plot.py

```diff
@@ -226,6 +226,17 @@ def test_sqlplot(tmp_empty, ip, cell):
     assert type(out.result).__name__ in {"Axes", "AxesSubplot"}
 
 
+@pytest.fixture
+def load_penguin(ip):
```

Comment:
We use the same code in other places. Let's move this fixture to [conftest.py](https://github.com/ploomber/jupysql/blob/80743a5cecb9eb63efa134dbefba6570b9a9eaf1/src/tests/conftest.py).